### DISCUSSION TITLE
Dalam t.test apakah hasil p.value nya selalu (default) bernilai 0,5 ?

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#536] [Q&A] [answered]
Dalam t.test yang pernah disampaikan dalam kelas PS didapatkan nilai p.value = 0,5 dan t.test yg coba saya lakukan p.value juga bernilai 0,5  sbb ;
<img width="514" alt="image" src="https://user-images.githubusercontent.com/123436963/217433489-f7a5bb38-93e8-4662-85d7-a0edeca3d0d5.png">

<img width="620" alt="image" src="https://user-images.githubusercontent.com/123436963/217433584-ba87c4af-a17f-40a8-8be8-b1dc367d4182.png">
 
Apakah nilai default dari p.value pada t.test adalah 0,5 ?
Terimakasih

**Comment by kevbow** [comment]
Dear @Micoalgo,

Terima kasih atas pertanyaan yang disampaikan, pertanyaan Anda sudah kami terima.
Harap untuk menunggu jawaban dari kami. 

Regards.

**Comment by Micoalgo** [comment, marked as answer]
Baik Terimakasih

**Reply by ranydwi** [reply]
Dear Pak @Micoalgo 

Terima kasih untuk pertanyaannya dan juga waktunya untuk menunggu. Saya akan coba untuk menjawab permasalahan yang Bapak hadapi.

1. Pada inclass yang Bapak maksud, sepertinya ada perbedaan jawaban yang Bapak dapat. Karena pada t-test di inclass, kita mempunyai nilai pengujian hipotesis sebagai berikut:
![image](https://user-images.githubusercontent.com/68055107/217481318-a1a6462a-639b-4df4-8513-fce030a78894.png)
> yang mana, untuk hal ini nilai `mu` yang kita masukan adalah nilai exact pengujian yang kita harapkan (yang kita masukan ke dalam hipotesis). Sehingga bukan merupakan nilai mean dari datanya.

2. Apakah nilai p-valuenya selalu default di 0.5? Jawabannya adalah **tidak**. 
> nilai p-value yang bapak dapatkan keduanya bernilai 0.5 karena nilai t-value = 0 (bisa bapak lihat dari output) hal ini merupakan sebuah kesalahan karena bapak salah memasukan parameter `mu`. Silahkan isi parameter `mu = 1385880` pada project yang bapak lakukan dan masukan parameter `mu = 215` pada inclass.

Semoga membantu :)
Terima kasih

**Comment by Micoalgo** [comment]
Terimakasih Jawabannya
